styles/sections/_hero.scss

```diff
@@ -41,29 +46,27 @@
   }
 }
 
-.date-header {
-  color: colors.$color-white;
-  font-weight: 500;
-  display: inline-block;
-  line-height: 0em;
-  padding: 0 0.25em 0.75em 0.25em;
-  background-color: colors.$color-teal-50;
-}
-
 #hero-scroll {
-  font-size: 10px;
+  font-size: 0.625rem; // 10px
   margin-top: auto;
   margin-bottom: 50px;
```

Comment:
I think with the addition of `line-height`, things look more cramped on mobile. I think we could remove some of the bottom margin from the scroll link (I think I added an arbitrary amount when I did the work), maybe make it like 16-20ish px? I think this will help make the h1 text less cramped with the nav bar too

main branch:
![Screen Shot 2023-04-06 at 3 32 21 PM](https://user-images.githubusercontent.com/62044293/230505709-ca9052aa-5d47-4e46-ae1c-d81cbd568f1c.png)
your branch:
![Screen Shot 2023-04-06 at 3 32 36 PM](https://user-images.githubusercontent.com/62044293/230505748-d6757a50-89f7-4a3f-b8f7-8a8227c58da0.png)

Reply:
good call